include/tscore/ink_memory.h

```diff
@@ -49,7 +49,11 @@
 #endif
 
 #if TS_HAS_JEMALLOC
+#if __has_include(<jemalloc/jemalloc.h>)
 #include <jemalloc/jemalloc.h>
+#else
+#include <jemalloc.h>
```

Comment:
Ee only need to have `#include <jemalloc.h>` in the code for compiling with `cmake`.    The include directory detection add in the subdir `jemalloc`.

However, with autotools we need to have the `#include <jemalloc/jemalloc.h>`